### DISCUSSION TITLE
yarn e2e GitHub actions are often broken

### DISCUSSION THREAD

**Discussion by Josh-Cena** [facebook/docusaurus#5132] [Q&A]
The Docusaurus repo's workflow often reports `/bin/sh: 1: docusaurus: not found`. (See https://github.com/facebook/docusaurus/runs/2969838005 for example. Rerunning the test by pushing an empty commit fixed this.) Is this a bug with the E2E actions workflow itself?

**Comment by slorber** [comment]
Also seen this but can't understand why this happens, unfortunately.

**Reply by Josh-Cena** [reply]
Mmm. Is there a convenient way to debug this? Looks like this is because the dependency is not properly installed before test starts (like having two jobs running in parallel).

**Reply by slorber** [reply]
if it was easy to debug I would have done it already 😅 

you can try running the e2e workflow locally and see if it can fail locally for some reasons, but this is time consuming
